brew.sh

```diff
@@ -26,6 +26,12 @@ brew install bash
 brew tap homebrew/versions
 brew install bash-completion2
 
+# Switch to using brew-installed bash as default shell
+if ! fgrep -q '/usr/local/bin/bash' /etc/shells; then
+  echo '/usr/local/bin/bash' >> /etc/shells;
```

Comment:
It was actually changed due to this comment - https://github.com/mathiasbynens/dotfiles/issues/656#issuecomment-216432628